.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description of Changes
+---
+Replace this text with an overview of your changes made in this Pull Request. Please use your best judgement here, do not be verbose to the point that you are giving an exact step-by-step of your workflow, but do not undersell the changes made. If this Pull Request addresses an open issue, you should reference that too.
+
+### Visual Sample
+---
+Replace this text with a media attachment or code test snippet that demonstrates the functionality of your changes. Please be mindful that GitHub is a platform for everyone, refrain from sending big attachments that could take a very long time to download for users with slow internet speeds.
+
+### Checklist
+---
+
```

Comment:
Might need a hint that you are supposed to put capital X in the brackets to make the check work

Reply:
I thought about this but evaluated that teaching markdown is out of scope, and you're also able to check the boxes in the preview and after submitting the PR. Can change if you think this is still necessary considering.

Reply:
No its okay